### DISCUSSION TITLE
Set Up Log Level DEBUG in API Gateway

### DISCUSSION THREAD

**Discussion by flower-of-the-bridges** [mia-platform/community#287] [Q&A] [answered]
### Which section?

Console

### Which is the version?

api-gateway:4.1.4

### Description

Hi

How can I change the log level of the nginx api-gateway and set the log level to debug?

**Comment by davidebianchi** [comment, marked as answer]
Hi @flower-of-the-bridges! It is not possible to change the mia-platform nginx log level by environment variable. Why do you need it?

**Reply by flower-of-the-bridges** [reply]
Hi @davidebianchi I'd like to see the complete request that the api-gateway receives from a client for debugging purposesù

**Reply by davidebianchi** [reply]
What do you mean with complete request? The api-gateway should already log the request path. Log some headers and body is not possible to log at the moment.
